### DISCUSSION TITLE
How to work with 3D asset

### DISCUSSION THREAD

**Discussion by coyzeng** [sugarme/gotch#124] [Q&A]
I want to use gotch to embedding 3D asset (obj format file) like https://github.com/jina-ai/executor-3d-encoder/blob/main/executor/models/encoder_pl.py

How to do it?

**Comment by sugarme** [comment]
@coyzeng,

Can you look at the `example` folder. There are examples there. If you get stuck at specific point, ask specific question so we can help you out.
